tsd/CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ if (TSD_USE_CUDA)
   find_package(CUDAToolkit REQUIRED)
 endif()
 
+if (USE_NEURAL_GRAPHICS_PRIMITIVES)
+  add_compile_options(-mf16c)
```

Comment:
Does this work on Windows? I just enabled CI (and added you to the project so CI should auto-run), so I suspect this won't compile on Windows. Will probably need platform-specific logic for the right flag(s) for the right compilers (gcc, clang, msvc).